ui/app/common/patient/services/patientService.js

```diff
@@ -32,12 +32,17 @@ angular.module('bahmni.common.patient')
 
         this.search = function (query, offset, identifier) {
             offset = offset || 0;
-            return $http.get(Bahmni.Common.Constants.bahmniSearchUrl + "/patient", {
+            var url = "/patient";
+            identifier = query;
+            url = url + "/lucene";
+            return $http.get(Bahmni.Common.Constants.bahmniSearchUrl + url, {
```

Comment:
Why not create another constant for the URL? "/patinet/lucene"

Reply:
modified